comfy_cli/workspace_manager.py

```diff
@@ -281,21 +281,51 @@ def get_comfyui_manager_path(self):
         if self.workspace_path is None:
             return None
 
-        # To check more robustly, verify up to the `.git` path.
-        manager_path = os.path.join(
-            self.workspace_path, "custom_nodes", "ComfyUI-Manager"
+        cached_manager_path = self.config_manager.get(
```

Comment:
Previously, we knew the absolute path of the comfyUI manager. With the new changes, the comfyUI manager directory name would change depending on which version you've installed (e.g., nightly vs. specific version). We need to figure out the directory of the ComfyUI Manager. 

Let's add comments explaining what this logic is.

Reply:
This logic may not work if we are caching the repository name for multiple workspace.